pkg/driver/postgres/postgres.go

```diff
@@ -115,10 +124,12 @@ func (drv *Driver) openPostgresDB() (*sql.DB, error) {
 		return nil, err
 	}
 
-	// connect to postgres database
-	postgresURL.Path = "postgres"
+	// connect to postgres database, unless this is a Redshift connection
+	if drv.databaseURL.Scheme != "redshift" {
+		postgresURL.Path = "postgres"
```

Comment:
If we don't change the root DB here, does that mean that `create` / `drop` / `dump` etc don't work at all?

In that case maybe it is better to give a specific error message rather than silently failing?

Reply:
On standard out-of-the-box Redshift, there is no database named `postgres`.

The [documentation](https://docs.aws.amazon.com/redshift/latest/dg/r_DROP_DATABASE.html) specifies that you cannot drop the `dev`, `padb_harvest`, `template0`, `template1`, or `sys:internal` databases, nor the current database.  So, for Redshift, we could try connecting to the `sys:internal` database in order to drop the target database, as long as it's not one of the listed databases.

I don't think we'll silently fail.  We should get an error from Redshift that the database cannot be dropped, most likely because it's the current database.

Reply:
@aterekhov-plr How important is it for dbmate to be able to drop/create the Redshift database?

Reply:
@amacneil How do you feel about this?  I like @aterekhov-plr's approach of moving forward as-is, and if anyone needs the functionality, they can open an issue and we can sort it out then?

Reply:
I don't think we need drop/create functionality for redshift, as long as the error message is clear.